src/blockchain/testnet_funds.cpp

```diff
@@ -151,26 +151,31 @@ Funds TestnetFunds() {
       P2WPKH(2975200 * UNIT, "e6f9018048751fc0cedeb19b9fbc5f1673659f8c"),
       P2WPKH(2945400 * UNIT, "b15202a43e089c5ca24159d8236af9f7f1b0d1e0"),
       P2WPKH(2916800 * UNIT, "a3f2b76e683276a4da2beeea42cf392e1348fd44"),
-      P2WPKH(10000000 * UNIT, "6927ce022826332f920b52cdb093a273f76afbaf"),
-      P2WPKH(10000000 * UNIT, "d58fcb388a30143b3bd2fcc07b0c87b552c7e79a"),
-      P2WPKH(10000000 * UNIT, "0277936f4315fdbda3d2854f0d0e770e1c9182e8"),
-      P2WPKH(10000000 * UNIT, "5b4c2e1892baf7b0d8961af9dcabaaa73da9491f"),
-      P2WPKH(10000000 * UNIT, "3cc7945a869d3ccd9f5f355d36d038fd065d31ea"),
-      P2WPKH(10000000 * UNIT, "f4ce59d1786e00a4fd74228c06a6dd374c50fffa"),
-      P2WPKH(10000000 * UNIT, "4c15d335c48feefa635a37f7d6cb34a09e992963"),
-      P2WPKH(10000000 * UNIT, "a0b2176374b54cd5d6408e35a1a10a6c24a1ff8d"),
-      P2WPKH(10000000 * UNIT, "ebe1dee4b462a321681505a9e1962ac364fe16ac"),
-      P2WPKH(10000000 * UNIT, "3f990e1d263e59af3ad472b69a6f3fc97c898969"),
-      P2WPKH(10000000 * UNIT, "2176bae014a27bb24903db14aa43afd9bff1f647"),
-      P2WPKH(10000000 * UNIT, "b7e787e8e8a65c8a2d45af03f96d6931d0d0ede1"),
-      P2WPKH(10000000 * UNIT, "724b37df95123d320a718426f2cd41743e84d968"),
-      P2WPKH(10000000 * UNIT, "779e3290852c10b51ddb022d361b76decd8f6751"),
-      P2WPKH(10000000 * UNIT, "cd9af4f42f742e1776546f2be71c815eff58be49"),
-      P2WPKH(10000000 * UNIT, "5382b0fbff427968dbc9acb718c9ada12ff6064c"),
-      P2WPKH(10000000 * UNIT, "2bee317e0b4775cfa26236aa8859b5298dd37c30"),
-      P2WPKH(10000000 * UNIT, "443ec5e6e8332c6376ea86ede6d0b3587777f660"),
-      P2WPKH(10000000 * UNIT, "6da5332cc4b6a84e6c6524bb4397b1e716cfca98"),
-      P2WPKH(10000000 * UNIT, "574e7e1b0bae9eb7a23eb4e90347b6ffa146010b"),
+      P2WPKH(8000000 * UNIT, "6927ce022826332f920b52cdb093a273f76afbaf"),
```

Comment:
Why did you also change these amounts?

Reply:
The total amount has to be the same as previously.